src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -652,66 +652,6 @@ function copyOptions() {
     feedbackRecipientUpdateVisibilityOptions($currRecipient);
 }
 
-function enableRow(elem, row) {
-    var $visibilityOptions = $(elem).closest('form').find('.visibilityOptions');
-    var $table = $visibilityOptions.find('table');
-    var $tdElements = $($table.children().children()[row]).children();
-    
-    if ($tdElements.parent().prop('tagName') === 'tr') {
-        return;
-    }
-    $tdElements.unwrap().wrapAll('<tr>');
-}
-
-function disableRow(elem, row) {
-    var $visibilityOptions = $(elem).closest('form').find('.visibilityOptions');
-    var $table = $visibilityOptions.find('table');
-    var $tdElements = $($table.children().children()[row]).children();
-    
-    if ($tdElements.parent().prop('tagName') === 'hide') {
-        return;
-    }
-    $tdElements.unwrap().wrapAll('<hide>');
-    $tdElements.parent().hide();
-}
-
-function feedbackRecipientUpdateVisibilityOptions(elem) {
```

Comment:
indeed, that's why need to rationalize out an interface with the feedback path options; alternatively, combine feedback path and visibility options